test/ListPredictionTest.cs

```diff
@@ -1052,6 +1069,9 @@ public void List_PluginSource_Acceptance()
                         NextLine,
                         TokenClassification.None, new string(' ', windowWidth)
                      )),
+                // `OnSuggestionDisplayed` should be fired for both predictors.
+                CheckThat(() => AssertDisplayedSuggestions(count: 2, predictorId_1, 56, 2)),
```

Comment:
56 here should also be MiniSessionid?

Reply:
Of course. My bad that I missed them :)

Reply:
I just pushed the changes. Could you please review again when you have time, thanks!